blocks/related-blogs/related-blogs.css

```diff
@@ -10,3 +10,128 @@
     color: #97f5f7;
     margin-bottom: 25px;
 }
+
+.related-list-item {
+    background-color: #fff;
+    padding: 10px 20px;
+    margin-right: 2%;
+    border: 0;
+    width: 28.5%;
+}
+
+ul.related-list {
+    list-style: none;
+    padding-left: 0;
+}
+
+.related-list {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: left;
+    margin-bottom: 25px;
+}
+
+.related-list-item:last-child {
+    margin-right: 0;
+}
+
+.teaser-link {
+    margin-bottom: 75px;
+}
+
+a.teaser-link {
+    text-decoration: none;
+}
+
+div.teaser {
+    display: flex;
+    height: 100%;
+    flex-direction: column;
+}
+
+h2.teaser-description {
+    font-size: 22px;
+    color: #2e308e;
+    font-weight: 400;
+    line-height: 1.25em;
+}
+
+.authorprofile-container {
+    padding-bottom: 35px;
+}
+
+.authorprofile-info {
+    display: inline-flex;
+    line-height: 1.4em;
+    flex-flow: column wrap;
+    vertical-align: top;
+}
+
+.authorprofile-name {
+    font-size: 14px;
+    font-weight: 700;
+    letter-spacing: -0.5px;
+    line-height: 1.4em;
+}
+
+.authorprofile-position {
+    font-size: 14px;
+    font-weight: 400;
+    letter-spacing: -0.5px;
+    line-height: 1.4em;
+}
+
+.authorprofile-image {
+    width: 35px;
+    height: 35px;
+    margin-right: 8px;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+img.nc-image {
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+div.nc-image-container {
+    max-width: 100%;
+    display: flex;
+    align-items: center;
+    position: relative;
+}
+
+div.related-button-row {
+    text-align: center;
+    color: #2e308e;
+}
+
+i.icon-wrapper {
+    height: 1em;
+    width: 1em;
+    display: inline-block;
+}
+
+.icon-wrapper svg {
+    pointer-events: none;
+    height: 100%;
+    width: 100%;
+    will-change: transform;
```

Comment:
Mozilla developer docs only recommend using will-change as a lasst resort to deal with existing performance problems, consider removing it (https://developer.mozilla.org/en-US/docs/Web/CSS/will-change).